src/core/frames/frame_controller.ts

```diff
@@ -412,11 +420,10 @@ export class FrameController
 
       frame.delegate.fetchResponseLoaded = (fetchResponse: FetchResponse) => {
         if (frame.src) {
-          const { statusCode, redirected } = fetchResponse
-          const responseHTML = frame.ownerDocument.documentElement.outerHTML
-          const response = { statusCode, redirected, responseHTML }
+          const { statusCode, redirected, response } = fetchResponse
+          const visitResponse = { statusCode, redirected, response }
```

Comment:
I'm not confident this change is consistent with the old logic where the `responseHTML` was pulled from the frame. I attempted to address that in [b248740](https://github.com/github/turbo/pull/1/commits/b24874019287d18707297746dd6876427975ecc7) but it seems to have broken more things than it fixed.